src/toil_scripts/batch_alignment/bwa_alignment.py

```diff
@@ -116,15 +129,20 @@ def run_bwa(job, inputs, ids):
     for fileStoreID, name in zip(ids, file_names):
         job.fileStore.readGlobalFile(fileStoreID, os.path.join(work_dir, name))
 
-    # Add read group line
-    rg = "@RG\\tID:{0}\\tLB:{1}\\tPL:{2}\\tPU:{3}\\tSM:{0}".format(inputs.uuid, inputs.library,
-                                                                   inputs.platform, inputs.program_unit)
+    rg = ''
+    if inputs.rg_line:
+        rg = inputs.rg_line
+    else:
+        rg = "@RG\\tID:{0}".format(inputs.uuid)
+        for tag, info in zip(['LB', 'PL', 'PU', 'SM'], ['library', 'platform', 'program_unit', 'uuid']):
+            if hasattr(inputs, info):
+                rg = rg + '\\t{0}:{1}'.format(tag, getattr(inputs, info))
```

Comment:
Oh, small nit, but while we're updating this line, we should add a comment to note that the `\t` has to have the `\` escaped because bwakit itself is expecting to get `\t` as two ASCII characters, not as a tab.